### DISCUSSION TITLE
Swagger page can't load (ERR_CERT_COMMON_NAME_INVALID)

### DISCUSSION THREAD

**Discussion by Xakim1c** [fastapi/fastapi#11513] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
def get_swagger_ui_html(
    *,
    openapi_url: str,
    title: str,
    swagger_js_url: str = "https://cdn.jsdelivr.net/npm/swagger-ui-dist@5/swagger-ui-bundle.js",
    swagger_css_url: str = "https://cdn.jsdelivr.net/npm/swagger-ui-dist@5/swagger-ui.css"
```

### Description

GET https://cdn.jsdelivr.net/npm/swagger-ui-dist@5/swagger-ui.css net::ERR_CERT_COMMON_NAME_INVALID       
GET https://cdn.jsdelivr.net/npm/swagger-ui-dist@5/swagger-ui-bundle.js net::ERR_CERT_COMMON_NAME_INVALID

### Operating System

Linux

### FastAPI Version

0.100.0

### Pydantic Version

1.10.11

### Python Version

Python 3.9.18

**Comment by amuzeev** [comment]
Дружище, под VPN (non-russian) работает. Что-то с нашим интернетом.

**Comment by deight93** [comment, marked as answer]
Check this issue.🥲
https://github.com/jsdelivr/jsdelivr/issues/18565

- Temporary fix
  - https://github.com/jsdelivr/jsdelivr/issues/18565#issuecomment-2089796648
